addons/website_sale/tests/test_website_sale_pricelist.py

```diff
@@ -558,6 +560,15 @@ def test_get_pricelist_available_geoip4(self):
             pls = self.website.get_pricelist_available(show_visible=True)
         self.assertEqual(pls, pls_to_return + current_pl, "Only pricelists for BE, accessible en website and selectable should be returned. It should also return the applied promo pl")
 
+    def test_get_pricelist_available_geoip5(self):
+        # Test get all available pricelists with geoip for a country not existing in any pricelists
+
+        self.website1_be_pl += self.env.user.partner_id.property_product_pricelist
+
+        with patch('odoo.addons.website_sale.models.website.Website._get_geoip_country_code', return_value=self.US.code):
+            pls = self.website.get_pricelist_available()
+        self.assertFalse(pls, "Pricelists specific to NL and BE should not be returned for US.")
```

Comment:
```suggestion
            pricelists = self.website.get_pricelist_available()
        self.assertFalse(pricelists, "Pricelists specific to NL and BE should not be returned for US.")
```
Unless the variable is reaaaally long (or you're in a limited scope, like lambda functions), no need to use a shortened version (it's clearer with the full name IMHO)

Reply:
Ecommerce is our scope, but the existing code is not always a good example, feel free to code differently if you feel like it's better (cleaner, faster, ...)